rm/rm-server/src/main/java/org/ow2/proactive/resourcemanager/selection/statistics/ProbablisticSelectionManager.java

```diff
@@ -292,4 +300,11 @@ private SelectionScript replaceBindings(SelectionScript script, Map<String, Seri
         }
     }
 
+    private String replaceBindingKeysByTheirValue(String scriptContent, Map<String, Serializable> binding) {
+        for (Map.Entry<String, Serializable> variableMapping : binding.entrySet()) {
+            scriptContent = scriptContent.replace(variableMapping.getKey(), variableMapping.getValue().toString());
```

Comment:
damn I forgot that possible NPE

Reply:
I guess you mean about `variableMapping.getValue()`. I have added the null check.